core/commands.py

```
@@ -832,6 +832,7 @@ def show_set_help_for_target(self, target: str) -> str:
                 ("api_id", "Telegram API ID（纯数字，从 my.telegram.org 获取）"),
                 ("api_hash", "Telegram API Hash（字符串，从 my.telegram.org 获取）"),
                 ("proxy", "代理地址（例如 http://127.0.0.1:7890 或 socks5://...）"),
+                ("debug_enabled_default", "QQ 发送诊断日志默认开关（true/false）"),
```

Ruff (0.15.12)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)